src/collection-preferences/content-display/content-display-option.scss

```diff
@@ -14,28 +14,16 @@
   @include styles.styles-reset;
   display: flex;
   align-items: flex-start;
-  padding-block: awsui.$space-xs;
-  padding-inline-start: 0;
-  padding-inline-end: awsui.$space-scaled-xs;
-  background-color: awsui.$color-background-container-content;
+  padding-block: awsui.$space-scaled-xs;
```

Comment:
in general, `scaled` variants should be used for vertical padding, hence the change here